addons/website/static/src/js/editor/widget_link.js

```diff
@@ -66,15 +66,18 @@ weWidgets.LinkTools.include({
                 $pageAnchor[0].querySelector('we-toggler').textContent = weTogglerText;
             };
             const urlWithoutHash = urlInputValue.split("#")[0];
-            wUtils.loadAnchors(urlWithoutHash, this.$editable[0].ownerDocument.body).then(anchors => {
-                for (const anchor of anchors) {
-                    const $option = $('<we-button class="dropdown-item">');
-                    $option.text(anchor);
-                    $option.data('value', anchor);
-                    $selectMenu.append($option);
-                }
-                always();
-            }).guardedCatch(always);
+            if (showAnchorSelector) {
```

Comment:
Should not the `always` definition and the `urlWithoutHash` definition be in that `if` as well?

Reply:
Yes indeed, it's adapted @qsm-odoo